docs/api/en/renderers/WebGLRenderer.html

```diff
@@ -187,6 +187,15 @@ <h3>[property:Object info]</h3>
 					<li>frame</li>
 				</ul>
 			</li>
+			<li>framebuffer:
```

Comment:
TBH, I find this notation confusing. When rendering to screen, you also render to a framebuffer. The default framebuffer.